### PR TITLE
Mapped some unmapped parameters in n/m/w/e/Entity

### PR DESCRIPTION
**Pull request by tmvkrpxl0**
I hope I mapped all unmapped ones there

### CODE REVIEW

File: data/net/minecraft/world/entity/Entity.mapping [hunk 1]
```diff
@@ -581,3 +604,17 @@ CLASS net/minecraft/world/entity/Entity
...
+	CLASS MoveFunction
+		METHOD accept (Lnet/minecraft/world/entity/Entity;DDD)V
+			ARG 1 entity
+			ARG 2 x
+			ARG 4 y
+			ARG 6 z
+	CLASS RemovalReason
+		METHOD <init> (Ljava/lang/String;IZZ)V
+			ARG 3 shouldDestroy
+			ARG 4 shouldSave
```
Comment (Harleyoc1) [inline data/net/minecraft/world/entity/Entity.mapping]: (field names)
```suggestion
			ARG 3 destroy
			ARG 4 save
```

File: data/net/minecraft/world/entity/Entity.mapping [hunk 2]
```diff
@@ -581,3 +604,17 @@ CLASS net/minecraft/world/entity/Entity
...
+	CLASS MovementEmission
+		METHOD <init> (Ljava/lang/String;IZZ)V
+			ARG 3 emitsSounds
+			ARG 4 emitsEvents
```
Comment (Harleyoc1) [inline data/net/minecraft/world/entity/Entity.mapping]: (field names)
```suggestion
			ARG 3 sounds
			ARG 4 events
```

File: data/net/minecraft/world/entity/Entity.mapping [hunk 3]
```diff
@@ -457,13 +468,17 @@ CLASS net/minecraft/world/entity/Entity
 		ARG 5 z
 	METHOD setDeltaMovement (Lnet/minecraft/world/phys/Vec3;)V
 		ARG 1 motion
+	METHOD setGlowingTag (Z)V
+		ARG 1 glowing
```
Comment (Harleyoc1) [inline data/net/minecraft/world/entity/Entity.mapping]: (field name)
```suggestion
		ARG 1 hasGlowingTag
```

File: data/net/minecraft/world/entity/Entity.mapping [hunk 4]
```diff
@@ -142,6 +149,8 @@ CLASS net/minecraft/world/entity/Entity
 		ARG 1 livingEntity
 	METHOD getEncodeId ()Ljava/lang/String;
 		COMMENT Returns the string that identifies this Entity's class
+	METHOD getExitPortal (Lnet/minecraft/server/level/ServerLevel;Lnet/minecraft/core/BlockPos;Z)Ljava/util/Optional;
+		ARG 3 isToNether
```
Comment (Harleyoc1) [inline data/net/minecraft/world/entity/Entity.mapping]: Could add mappings for the `level` and `pos` params.

**Review by Harleyoc1 — CHANGES_REQUESTED** [review]

File: data/net/minecraft/world/entity/Entity.mapping (hunk 4 above)
Comment (tmvkrpxl0) [reply data/net/minecraft/world/entity/Entity.mapping]: Done, but it seems like I missed some, ones that are red in enigma should be ones that I missed

Comment (tmvkrpxl0) [reply data/net/minecraft/world/entity/Entity.mapping]: I ignored one that are not "float1" or "integer1" but sometimes named parameters had those names. I should recheck every parameter

**Comment by tmvkrpxl0** [comment]
Ok, I think I remapped everything, as I don't see red parameters in enigma

File: data/net/minecraft/world/entity/Entity.mapping [hunk 5]
```diff
@@ -475,11 +554,15 @@ CLASS net/minecraft/world/entity/Entity
 		ARG 1 x
 		ARG 3 y
 		ARG 5 z
+	METHOD setPacketCoordinates (Lnet/minecraft/world/phys/Vec3;)V
+		ARG 1 coordinates
```
Comment (Harleyoc1) [inline data/net/minecraft/world/entity/Entity.mapping]: Again, try to keep the names consistent with the official fields/methods (match field names where possible). 
```suggestion
		ARG 1 packetCoordinates
```

**Review by Harleyoc1 — CHANGES_REQUESTED** [review]
One other minor comment.

**Review by Harleyoc1 — APPROVED** [review]

**Review by sciwhiz12 — APPROVED** [review]

**Pull request merged**